### DISCUSSION TITLE
[memo] Local News Livecast Server List by Owner

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#134] [Show and tell]
- Not Tokened
1. Uplynk
   ABC Owned Stations , E.W.Scripps Company, Sinclair
2. Dai
    CBS Owned Stations (CBS News Live Locals)
Others
NBC Owned Stations (akamai) TEGNA (Original) COX Media Group (amagi)

- Tokened
1. Anvato
Nextar Media Group, CBS Owned Stations (Non-CBS News Live),FOX Owned Stations
2. Uplynk
Hearst Television (pbs=[token])

2.aws syncback
Gray Television

**Comment by BellezaEmporium** [comment]
> * Not Tokened
> 
> 
>     1. Uplynk
>        ABC Owned Stations , E.W.Scripps Company , Nextstar
>        Others
>        NBC Owned Stations (akamai)
> 
> 
>     * Tokened
> 
> 
>     1. Anvato
>        Nextar Media Group, CBS Owned Stations (Non-CBS News Live),FOX Owned Stations
> 
> 
> 2.aws syncback Gray Television

I know how to work with Anvato-tokened links. Like another website, i'll work on a patch and send it to dct-infra for deploying to free.fullspeed.
